assets/javascripts/spoiler.js

```diff
@@ -1,6 +1,7 @@
 (function($) {
 
   var isIE = /*@cc_on!@*/false || document.documentMode,
+      isFirefox = false || navigator.userAgent.indexOf("Firefox") > 0,
```

Comment:
I'd rather we use `var isFirefox = typeof InstallTrigger !== 'undefined';` to detect firefox